e2e/tests/manage.spec.ts

```diff
@@ -261,15 +261,15 @@ test('Move a booking', async ({ page, person, user }) => {
   await placementPage.showsBedMoveLoggedMessage()
 })
 
-test('View all out of service beds', async ({ page, user }) => {
-  // Given I have the 'future_manager' role
-  await setRoles(page, user.name, ['future_manager'])
-  // And I am on the dashboard page
-  const dashboard = await visitDashboard(page)
-
-  // And I click the 'View out of service beds' tile
-  dashboard.clickOutOfServiceBeds()
-
-  // Then I am taken to the out of service beds page
-  await OutOfServiceBedsPage.initialize(page, 'View out of service beds')
-})
+// test('View all out of service beds', async ({ page, user }) => {
```

Comment:
could we use the `.skip` annotation here rather than commenting out? 

https://playwright.dev/docs/test-annotations#skip-a-test

That might be clearer in terms of what the runner reports? i.e. that we see explicitly what we we didn't run...